### ISSUE TITLE
not working?

### ISSUE THREAD

**Issue by tiilopes** [icecreamdatabase/IceCreamDataBase#104]
TTS is working?

**Comment by cineafx** [comment]
Yes. The tts provider is blocking third party access due to the size of the bot. We are looking into replacements but this will most likely take a few weeks or even months. We are terribly sorry for the inconvenience.

**Comment by cineafx** [comment]
I'm going to keep this issue open so other people can find the answer.
